OrbitCore/CMakeLists.txt

```diff
@@ -215,17 +215,16 @@ target_sources(OrbitCoreTests PRIVATE
 if(NOT WIN32)
   # TODO: Enable ElfFileTests.cpp for all platforms once we have llvm support on Windows.
   target_sources(OrbitCoreTests
-    PRIVATE ElfFileTests.cpp
-            OrbitModuleTest.cpp)
-  target_link_libraries(OrbitCoreTests PRIVATE  ${llvm_libs})
+    PRIVATE OrbitModuleTest.cpp)
 endif()
 
 target_link_libraries(
   OrbitCoreTests
   PRIVATE OrbitCore
           GTest::Main
           llvm_object::llvm_object
-          abseil::abseil)
+          abseil::abseil
+          ${llvm_libs})
```

Comment:
`${llvm_libs}` is from the old vcpkg-days and shouldn't be there. My apologies, I probably missed to remove it.
It would be great if you could remove it now.

Reply:
done